edge/services/mms/prod-css-write.sh

```diff
@@ -0,0 +1,68 @@
+#!/bin/bash
+
+# Create a new "objectID" (of a specified "objectType" at a specified "version)
+# in the production (cloud) CSS.
+
+# User must have `HZN_CSS_URL` set in their environment
+if [ -z "$HZN_CSS_URL" ]; then
+    echo "ERROR: \"HZN_CSS_URL\" must contain the cloud CSS URL."
+    exit 1
+fi
+
+# Local CSS REST API endpoint (i.e., the base URL) -- NOTE: use `http`!
+CSS_ENDPOINT="${HZN_CSS_URL}/api/v1"
+
+# User must have `HZN_ORG_ID` and `HZN_EXCHANGE_USER_AUTH` set too
+if [ -z "$HZN_ORG_ID" ]; then
+    echo "ERROR: \"HZN_ORG_ID\" must contain your organization name."
+    exit 1
+fi
+if [ -z "$HZN_EXCHANGE_USER_AUTH" ]; then
+    echo "ERROR: \"HZN_EXCHANGE_USER_AUTH\" must contain your credentials."
+    exit 1
+fi
+
+# Construct the auth string
+AUTH="${HZN_ORG_ID}/${HZN_EXCHANGE_USER_AUTH}"
+
+# Metadata for the object being created
+objectType="$1"
+objectID="$2"
+version="1.0.0"
+description=""
+
+# Which Edge Nodes to target ("" is the wildcard for all IDs/types)
+destinationID=""
+destinationType=""
```

Comment:
In a prod environment, I think you want them setting the destination type to the pattern so that these objects are not getting onto all the nodes.

Reply:
I added a detailed comment in the file to clarify this.